src/components/TracksTable.vue

```diff
@@ -77,7 +78,7 @@
       </template>
       <template v-slot:item.album_id="props">
         <RouterLink :to="{ name: 'album', params: { id: props.value } }">
-          {{ albums[props.value].title }}
+          {{ albums[props.value] ? albums[props.value].title : "" }}
```

Comment:
Is this not always present?

Reply:
No, it could always be that a track is loaded before the album is. But I'll extract this to a separate PR, since this is a bug fix